### PR TITLE
[Gui] New Icons and rearrangement of Draw Style menu

### PR DESCRIPTION
**Pull request by carlopav**
Just an experiment, didn't try to compile
Gui: Changed icons in Draw Style menu
And also saved old icons in pure svg.

### CODE REVIEW

**Comment by triplus** [comment]
I see you have changed the command object names. In that case i will have to update the macros:

https://forum.freecadweb.org/viewtopic.php?f=3&t=14336&start=50#p364692

P.S. Not a big deal, just wondering if it is necessary to change the object names in addition to changing the icons.

**Comment by carlopav** [comment]
> 
> 
> I see you have changed the command object names.
> 

Hmmm this was not my intention, I just wanted them to appear in the menu in the order from the most simple (points) to the most complete (flat lines), so I thought to re-order them in every list of the file, but I probably did something wrong...

**Comment by triplus** [comment]
My bad. You didn't use different object names, you have just moved them around a bit. In that case the mentioned macros should continue to work just fine.

Sorry for the noise.

**Comment by carlopav** [comment]
Anyway, if you know someone who can test it and correct whatever needed, feel free to add him to the conversation :)

File: src/Gui/CommandView.cpp [hunk 1]
```diff
@@ -597,33 +597,33 @@ Gui::Action * StdCmdDrawStyle::createAction(void)
...
-    a5->setIcon(BitmapFactory().iconFromTheme("DrawStyleWireFrame"));
-    a5->setObjectName(QString::fromLatin1("Std_DrawStyleHiddenLine"));
+    a5->setIcon(BitmapFactory().iconFromTheme("DrawStyleShaded"));
+    a5->setObjectName(QString::fromLatin1("Std_DrawStyleShaded"));
     a5->setShortcut(QKeySequence(QString::fromUtf8("V,6")));
     QAction* a6 = pcAction->addAction(QString());
-    a6->setCheckable(true);
-    a6->setIcon(BitmapFactory().iconFromTheme("DrawStyleWireFrame"));
-    a6->setObjectName(QString::fromLatin1("Std_DrawStyleNoShading"));
+    a1->setCheckable(true);
```
Comment (triplus) [inline src/Gui/CommandView.cpp]: Likely this should be a6 instead of a1.

**Comment by triplus** [comment]
In general it looks OK to me. I added a note, as one line needs changing. I see you have already discussed the proposed change on the forum.

P.S. After you will update the PR best to change the title to indicate this PR is ready to be merged.

**Comment by carlopav** [comment]
Good catch! i updated and rebased, but the point is that i didn't try to compile it... do you think it's safe to merge it anyway?

**Comment by sgrogan** [comment]
Travis will compile it for you. https://travis-ci.org/FreeCAD/FreeCAD/builds/659359801?utm_source=github_status&utm_medium=notification

**Comment by triplus** [comment]
One additional thing you should do is to add newly introduced icon (DrawStyleHiddenLine.svg) to:

https://github.com/FreeCAD/FreeCAD/blob/master/src/Gui/Icons/resource.qrc

The rest looks OK to me.

Reference:
https://forum.freecadweb.org/viewtopic.php?p=371686#p371686

**Comment by carlopav** [comment]
Done, Thanks @triplus and @sgrogan

**Pull request merged**